maven-core/src/main/java/org/apache/maven/internal/aether/LoggingRepositoryListener.java

```diff
@@ -96,12 +96,7 @@ public void artifactDescriptorInvalid(RepositoryEvent event) {
         buffer.append("The POM for ");
         buffer.append(event.getArtifact());
         buffer.append(" is invalid, transitive dependencies (if any) will not be available");
-
-        if (logger.isDebugEnabled()) {
-            logger.warn(buffer + ": " + event.getException().getMessage());
-        } else {
-            logger.warn(buffer + ", enable debug logging for more details");
-        }
+        logger.warn(buffer + ": " + event.getException().getMessage());
```

Comment:
Maybe one should probe if debug is enabled and pass `e` otherwise?

Reply:
I don't want to _increase_ output, I want to make it more useful :stuck_out_tongue:

Reply:
Even in debug level?

Reply:
WDYM "even in debug level"? This listener is triggered when there is invalid model reported... stack trace WHERE is totally irrelevant, it is much much import WHY and WHICH model was invalid.

Reply:
In mean, it other words we _know_ from where even comes: from model builder and will always come from model builder as this event is triggered from it when it reports invalid model. So zero information (or zero value) has the stack trace.

Reply:
Makes sense.